portfolio.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF">
+  <title>portfolio</title>
+  <link rel="stylesheet" type ="css" href="stylesheets/styles.css">
+</head>
+
+<body>
+  <header>
+    <nav>
+      <ul>
+        <li><a href="blog/blog-home.html">blog</a></li>
+        <li><a href="about.html">about me</a></li>
+        <li><a href="portfolio.html" class="current-page">portfolio</a></li>
+        <li class="home"><a href ="index.html">home</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <section class="section-background">
+    <h2>portfolio</h2>
+    <hr class="section-break">
+
+    <aside>
```

Comment:
I think you can do these a bit more semantically. Consider wrapping each entry (including the img) in an <article> tag, and wrapping the img in a <figure> tag.